aws/table_aws_codepipeline_pipeline.go

```diff
@@ -0,0 +1,257 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/codepipeline"
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsCodepipelinePipeline(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_codepipeline_pipeline",
+		Description: "AWS Codepipeline Pipeline",
+		Get: &plugin.GetConfig{
+			KeyColumns:        plugin.AllColumns([]string{"name", "region"}),
```

Comment:
Do we need region for GET?
Since we are not using this value anywhere

Reply:
Removed the region from GetConfig